### ISSUE TITLE
private db access disabled for this account

### ISSUE THREAD

**Issue by robflate** [boredazfcuk/docker-icloudpd#63]
Anyone come across the following error in the logs?

`private db access disabled for this account.  Please wait a few minutes then try again.  The remote servers might be trying to throttle requests. (ACCESS_DENIED)`

I started getting ``failed to download new files`` errors in the logs so I updated my 2FA cookie. Maybe Apple have just temporarily throttled requests after the initial failures?

**Comment by boredazfcuk** [comment]
Hi,

Yes this is likely due to throttling. I saw this a lot when I set my container to sync every hour. Set it back to 24hrs and it works fine.

If you're hitting the servers a lot, they will throttle the connection. If the container fails to connect, it will wait a couple of minutes before trying again. It may mean that the retries are what's locking you out.

Had your cookie expired? It should fire of a warning about that, or disable synchronisation at the very least. Hitting the icloud server every time, when it knows it has an expired 2FA token shouldn't happen... Maybe it is something for me to check.
